### ISSUE TITLE
Not a support double/redouble but showing 18-19

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#708]
06CBE47966E2161ED9848AFF8F91
![image](https://github.com/user-attachments/assets/e9008a7a-f700-456c-b05d-3db9157d32e7)

Had to force East to bid, pass with 11 HCP after partners overcall is to conservative,

**Comment by EdwardPiwowar** [comment]
Pass with 7 trumps is ok for me.

**Comment by ThorvaldAagaard** [comment]
The problem is once again, that after a double BBA tend to Pass in the hope to describe the hand later, and that is not a good style
![image](https://github.com/user-attachments/assets/13c08ea4-1aa0-4d61-8c9f-331f7713992d)

So after Double pass, but after Pass 1N, why the difference?

**Comment by EdwardPiwowar** [comment]
The score for 1SX or an opponent's play is potentially higher than for your own play.

**Comment by ThorvaldAagaard** [comment]
Yes, that is the common problem, not only here, but in many other situations. Pass to a T/O double hoping to play there is normally not good bridge, unless you play against weak opponents.
